lib/avo/fields/key_value_field.rb

```diff
@@ -15,10 +11,10 @@ def initialize(id, **args, &block)
 
         hide_on :index
 
-        @key_label = args[:key_label].present? ? args[:key_label].to_s : I18n.translate('avo.key_value_field.key')
-        @value_label = args[:value_label].present? ? args[:value_label].to_s : I18n.translate('avo.key_value_field.value')
-        @action_text = args[:action_text].present? ? args[:action_text].to_s : I18n.translate('avo.key_value_field.add_row')
-        @delete_text = args[:delete_text].present? ? args[:delete_text].to_s : I18n.translate('avo.key_value_field.delete_row')
+        @key_label = args[:key_label].to_s if args[:key_label].present?
```

Comment:
Do we **need** to cast to string in this case?

```suggestion
        @key_label = args[:key_label] if args[:key_label].present?
```